Gemfile

```diff
@@ -4,6 +4,9 @@ gemspec
 
 gem 'gettext', '>= 3.1.3', '< 4.0.0'
 
+# FFI 1.17 needs rubygems 3.3.22+, which is Ruby 3.0+ only
+gem "ffi", "<1.17" if RUBY_VERSION < '3.0'
```

Comment:
Shouldn't we rather use:

```suggestion
gem "ffi", "<1.17" if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('3.0')
```

?

Reply:
Nvm, I think it's not available during Gemfile evaluation?..

Reply:
Technically, maybe.
But we've used the exact same notation in other places (https://github.com/theforeman/foreman/blob/develop/Gemfile#L55) and it works fine.